ansible_roles/init/tasks/main.yml

```diff
@@ -20,8 +20,8 @@
 - name: Install SELinux bindings for Ansible
   yum: pkg=libselinux-python state=installed
```

Comment:
we should stick with one style, and i recommend the bellow YAML format. This would be also aligned with latest ansible standards int PNT